features/org.wso2.carbon.osgi.feature/pom.xml

```diff
@@ -94,11 +94,11 @@
             <artifactId>org.apache.felix.gogo.runtime</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.wso2.org.ops4j.pax.logging</groupId>
+            <groupId>org.ops4j.pax.logging</groupId>
```

Comment:
We need to check the diff of [wso2 fork](https://github.com/wso2/wso2-pax-logging/pull/10). I think we need to keep this and release the fork accordingly

Reply:
Updated with a release in the wso2 pax logging